narratives/ncov_sit-rep_2020-04-24.md

```diff
@@ -0,0 +1,259 @@
+* [Updates for Senegal](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-24?n=X).
+* [Updates for Ghana](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-24?n=X).
+* [Updates for Algeria, Gambia, Nigeria, and South Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-24?n=X).
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-24?n=X).
+* [FAQ & common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-24?n=X).
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-24?n=X).
+
+<!-- This is right-side text -->
+```auspiceMainDisplayMarkdown
+# Executive summary
+
+We analyzed 1,300 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading locally. In this week's updates, we report:
+<br><br>
+* Many separate introductions of SARS-CoV-2 to the African continent, primarily from Europe (although sampling bias may contribute to this inference)
+<br><br>
+* At least 7 introductions to the DRC  
+<br><br>
+* 2 separate transmission chains circulating in Kinshasa, DRC  
+<br><br>
+* Of 8 introductions to Senegal, at least 2 seeded local transmission chains in Dakar and Touba
+<br><br>
+* Local transmission in Greater Accra, Ghana, after 7 introductions to this region
+<br><br>
+* Scattered introductions to Algeria, Gambia, Nigeria, and South Africa
+```
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 2-->
+# [COVID-19 Resources](https://nextstrain.org/ncov/africa/2020-04-23?d=tree&p=full&legend=closed&f_region=Africa)
+We've prepared some resources that will make interpreting the data we present in this narrative easier.
+#### Nextstrain Resources  
+* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).  
+* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
+* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).
+
+#### External Resources  
+* [Ask a Scientist & FAQs](https://covid19.fas.org/l/en).
+* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
+* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+* [The NYTimes COVID-19 coverage](https://www.nytimes.com/news-event/coronavirus).
+<!-- There is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text -->
+# [A note on sampling](https://nextstrain.org/ncov/africa/2020-04-23?d=map&f_region=Africa&p=full)
+We currently have sequences from samples taken in 6 continents, including 7 African countries. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
```

Comment:
```suggestion
We currently have sequences from samples taken in 6 continents. In Africa, we have samples from 7 countries. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
```
I just think the orig wording is a little weird, it makes it sound like the continents include 7 African countries? Maybe it's just me! 😆